### ISSUE TITLE
can't follow world creation tutorial

### ISSUE THREAD

**Issue by fcseidl** [osrf/subt#949]
There are several problems here, but the most important is at the end; see my TL;DR below. 

I'm trying to modify one of the finals practice worlds to put more artifacts near the starting position, to facilitate collecting training images of the new artifacts. I'm attempting to follow the instructions provided [on the SubT Wiki](https://github.com/osrf/subt/wiki/World-Creation-Tutorial). 

**Problem 1:** I used the command ```ign launch competition.ign worldName:=finals_practice_01 circuit:=finals``` in hopes of using the drag to simulator feature from the tech repo as well as the ignition model manipulation tools to add new artifacts. This should be possible according to the tutorial, but in fact only the starting room loads in the simulator window and the command line complains

[ERROR] [1622658223.678880818]: Cannot operate without robot_names set

Oh well, I'll just add a robot. Running ```ign launch competition.ign worldName:=finals_practice_01 circuit:=finals robotName1:=Xa robotConfig1:=SSCI_X4_SENSOR_CONFIG_1``` allows me to load the full world, and add a fire extinguisher to the first tunnel, while a UAV sits idly by in the starting room. Including a robot shouldn't be necessary for this, but it seems to be.

**Problem 2:** I can't validate my .dot file. I saved the world as ```~/subt_ws/src/subt/subt_ign/worlds/test_world.sdf```, and remade using catkin. The visibility graph outputted by ```rosrun subt_ign dot_generator ~/subt_ws/src/subt/subt_ign/worlds/test_world.sdf``` looked reasonable, although it was proceeded by a long list of Unrecognized model type[] lines. The [Connections Validator tutorial](https://github.com/osrf/subt/wiki/Connections%20Validator%20Tool) says to run ```ign launch -v 4 validate_connections.ign worldName:=test_world.sdf``` but this command just spins its wheels. The world creation tutorial also offers a non-GUI method, which runs an executable from the ```~/subt_ws/install/bin/``` folder. This folder does not exist!

**Problem 3 (most severe):** In case I could get away without validating the .dot file, I carried on to generating the .dat. The command to do this, ```ign launch visibility.ign worldName:=test_world```, also spins its wheels for over 30 minutes without producing any output, and I don't see the promised ```~/subt_ws/install/share/subt_ign/worlds/test_world.dat``` file being created. Predictably, trying to load ```test_world``` in the simulator now causes it to complain about being unable to find the visibility data:

[Err] [VisibilityTable.cc:57] Unable to find full path for[test_world]
[Err] [VisibilityRfModel.cc:44] Unable to load visibility table data files
[Err] [GazeboFactory.cc:205] Factory service call timed out.

**TL;DR:** the world creation tutorial on the wiki includes several commands which do not work as the tutorial leads one to expect. Most critically, the ignition command to generate the visibility .dat file does not seem to halt.

**Comment by iche033** [comment]
> Problem 1: 

* Updated the wiki to mention that for certain world that contains levels, the user can specify `levels:=false` to disable levels and see the whole world.

> Problem 2:

* Added info about the `Unrecognized model type` msg. Most of them should be safe to ignore.
* Updated path to `validate_connections`. 

> Problem 3:

* Updated tutorial to specify `-v 4` flag to get verbose output, which may help identify what the issue is, i.e. `ign launch -v 4 visibility.ign worldName:=my_new_world`
* The worlds have grown much larger since that section of the tutorial was written so it can take quite some time to generate the dat file depending on the user's machine. I've made a note in the tutorial. For `finals_practice_01`, it takes more than 1 hour on my laptop (`ign launch -v 4 visibility.ign worldName:=finals_practice_01`). However, you should at least see a console output like `xx %`, to get an idea of the progress. If not, then check if there are any error msgs printed with the `-v 4` flag.
